rules-tests/DowngradePhp70/Rector/FuncCall/DowngradeDirnameLevelsRector/DowngradeDirnameLevelsRectorTest.php

```diff
@@ -1,33 +1,33 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Rector\Tests\DowngradePhp70\Rector\FuncCall\DowngradeDirnameLevelsRector;
-
-use Iterator;
-use Rector\Testing\PHPUnit\AbstractRectorTestCase;
-use Symplify\SmartFileSystem\SmartFileInfo;
-
-final class DowngradeDirnameLevelsRectorTest extends AbstractRectorTestCase
-{
-    /**
-     * @dataProvider provideData()
-     */
-    public function test(SmartFileInfo $fileInfo): void
-    {
-        $this->doTestFileInfo($fileInfo);
-    }
-
-    /**
-     * @return Iterator<SmartFileInfo>
-     */
-    public function provideData(): Iterator
-    {
-        return $this->yieldFilesFromDirectory(__DIR__ . '/Fixture');
-    }
-
-    public function provideConfigFilePath(): string
-    {
-        return __DIR__ . '/config/configured_rule.php';
-    }
-}
\ No newline at end of file
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Tests\DowngradePhp70\Rector\FuncCall\DowngradeDirnameLevelsRector;
+
```

Comment:
Btw, what has changed here? The file looks identical. EOLs?

Reply:
that's seems spacing and eol